app/views/child_benefit_tax/_results.html.erb

```diff
@@ -1,7 +1,7 @@
 <% results_html = capture do %>
   <div class="results-estimate">
-    <h3>Child Benefit received</h3>
-    <p class="numbers">
+    <h3 class="govuk-heading-m">Child Benefit received</h3>
+    <p class="govuk-!-font-size-80 govuk-!-font-weight-bold">
```

Comment:
I'm wondering if we shall keep a class for this instead of using overwrite classes?

Reply:
I considered that but figured it was just adding unnecessary CSS to the page weight. Is there a reason why we'd do a custom class for this instead?

Reply:
Overwrites are meant to be used as ultimate resorts as they generate `!important` properties.

Reply:
Ah, I see. Are there non-override classes I can use instead?

Reply:
There don't seem to be any non-override classes, unless I've missed them. I try to avoid important where I can and I see the intent of these classes but since they're already in the stylesheet it seems like we might as well use them rather than add more classes. Not sure. It's an interesting topic, at least!

Reply:
No. The idea behind `govuk-frontend` model is not to use classes to compose styles for elements/components, but to define new classes (with semantic, meaningful names instead of describing what they do at a certain point) that share the same mixins (in this case would be `govuk-font`) — so that composition happens at the SCSS level instead of the browser level. This provides a way for an element/component to have a single source for styles and to  remain confident that making a change to a class only affects the elements we want.

Reply:
I've just tried to add a class for this and it turns out I need the overrides after all - this element is influenced by govspeak styles. @alex-ju are you happy to approve?

Reply:
oh, govspeak 🤦‍♂sure!

Reply:
Yeah, I only noticed when I tried to change it 😁 

Thanks!